src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -33,8 +33,41 @@ $(document).ready(function () {
     updateConstSumMessages();
 
     prepareRubricQuestions();
+    
+    prepareMCQQuestions();
 });
 
+// Prepare mcq questions for answering by user
+function prepareMCQQuestions() {
+	// Get index of mcq question
+	var mcqQuestionNums = getQuestionTypeNumbers("MCQ");
+
+	for (var i = 0; i < mcqQuestionNums.length; i++) {
+		var qnNum = mcqQuestionNums[i];
+
+		// Get number of options for certain mcq question
+		var optionNums = $("[name^='responsetext-" + qnNum + "']").length;
+		for (var k = 0; k < optionNums; k++) {
+			// Get specific option for that question
+			var radioInput = $("[name^='responsetext-" + qnNum + "-" + k + "']");
+			radioInput.prop("checked", false);
+
+			// Toggle radio button if it was checked
+			radioInput.click(function(event) {
+				if ($(this).data("waschecked")) {
+					$(this).prop('checked', false);
+					$(this).data('waschecked', false);
```

Comment:
There are answers that do not use that. See: http://jsfiddle.net/Se9DL/729/
It may be better to not change the values/property as that would change the UI, which would affect tests.
Do check that the tests are passing for this pull request and write tests that ensure the deselecting of radio buttons work.